### ISSUE TITLE
3ds root folder not loading

### ISSUE THREAD

**Issue by MarioMaster100** [mashers/3ds_hb_menu#93]
I placed the files from https://github.com/mashers/3ds_hb_menu/raw/master/launcher.zip in the root of the sd card and booted up to homebrew launcher. It's a USA 2DS firmware version 10.5.0-30U . I hacked it via browserhax http://yls8.mtheall.com/3dsbrowserhax.php and I have menuhax installed. After booting to gridlauncher I clicked on the folder icon to load the 3ds directory but after loading it, nothing shows up from it. Before I installed this launcher it showed my 3ds folder contents on the original launcher.

**Comment by mashers** [comment]
Not sure what's happening there TBH. Does the content of the 3ds folder still show in the original launcher? What about if you try a different entrypoint?

**Comment by MarioMaster100** [comment]
Yeah the 3ds folder contents shows up in the original launcher when I copy the old file over the gridlauncher one. I think menuhax was the only available software hack for my firmware.

**Comment by mashers** [comment]
Can you post a screenshot of your SD card contents please?

**Comment by MarioMaster100** [comment]
http://imgur.com/TXtrWMe and http://imgur.com/81UONFy

**Comment by mashers** [comment]
Thanks. There's no gridlauncher folder there, and your boot.3dsx is 486k, which is far too small to be gridlauncher. Are you actually using gridlauncher? What do you see when you start HBL?

**Comment by MarioMaster100** [comment]
Oh well I'd been using my 2DS to run homebrew stuff so I put back the old HBL and removed the gridlauncher folder so I could access my 3ds folder again.

**Comment by mashers** [comment]
Ok, what does the inside of your 3ds folder look like?

**Comment by MarioMaster100** [comment]
3ds folder https://imgur.com/JFBbBKP
